### ISSUE TITLE
Question: Duplicate link text in the title attribute

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag-act#434]
Is it a fail of any WCAG SC where the title attribute text is a duplicate of the visible link text? Or is this just a recommend 'Not To Do' best practice issue?

For example `<a href="/privacy-policy" title="Privacy Policy">Privacy Policy</a>`

Thank you

**Comment by LaurenceRLewis** [comment]
Closing because screenreaders no longer announce duplicate links
